platform-definition.yaml

```diff
@@ -435,6 +440,9 @@ platform_definition:
       'org.springframework.data:spring-data-gemfire':
          - 'org.springframework.shell:spring-shell'
          - 'org.iq80.snappy:snappy'
+      'org.springframework.data:spring-data-geode':
+         - 'org.springframework.shell:spring-shell'
```

Comment:
Is this still needed? From what I've seen, I'm not sure that it is.

Reply:
Well, yes and no.  _Spring Shell_ is only needed at runtime, and only if the user is running one of the nodes in the GemFire cluster as a Management node (or a Manager).  Effectively for GemFire/Geode's shell tool to be able to connect to and inspect the cluster (possibly "manage" it) the Management Service needs to be running.  Effectively, that node then takes on the role of a "Manager" in the cluster allowing any JMX-based client (including JConsole, JVisualVM, etc) to connect and inspect/manage the cluster.  Since the Management Commands were built on Spring Shell's Command infrastructure, basically the Manager cannot function properly, to manage the cluster without _Spring Shell_ on the CLASSPATH.

Reply:
Sorry, I don't understand how the answer can be "yes and no". If `org.springframework.shell:spring-shell` is a direct dependency of `org.springframework.data:spring-data-geode` then this is needed. If it is not a direct dependency of  `org.springframework.data:spring-data-geode` then it is not needed.

Reply:
Well, it is an indirect dependency provided by Pivotal GemFire/Apache Geode (pertaining to Management) that enables certain features and (management) capabilities in _Spring Data Geode/GemFire_ when managing Apache Geode/Pivotal GemFire (remotely, even).

So, I am not certain what the problem is, or your concern is in particular, since the [`spring-shell` dependency](https://github.com/spring-projects/spring-data-geode/blob/2.0.x/pom.xml#L245-L256) is a ["test" dependency](https://github.com/spring-projects/spring-data-geode/blob/2.0.x/pom.xml#L249) as is the ["snappy" dependency](https://github.com/spring-projects/spring-data-geode/blob/2.0.x/pom.xml#L238-L243), as you instructed in [SGF-712]().

Reply:
Ok, so it is not a direct dependency. Therefore, this is redundant. Can you please remove it?